fetch-artifact

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
```

Comment:
There's no python in this repo yet I don't think, I would keep it that way unless it's __really__ needed, if we're needing to wrap oras let's try and do it with a bash script

Reply:
Just pushed a new (much simplified) bash script to replace it. Turns out Fedora comes with a super old Oras version that was preventing me from using the `--platform` option.